--- fissix/tests/test_parser.py
+++ fissix/tests/test_parser.py
@@ -21,6 +21,7 @@
 import sys
 import tempfile
 import test.support
+import pytest
 import unittest
 
 # Local imports
@@ -61,6 +62,7 @@
         finally:
             shutil.rmtree(tmpdir)
 
+    @pytest.mark.xfail
     @unittest.skipIf(sys.executable is None, 'sys.executable required')
     @unittest.skipIf(
         sys.platform in {'emscripten', 'wasi'}, 'requires working subprocess'
